openapi-v1.yaml

```diff
@@ -5,8 +5,25 @@ info:
   description: Security Token Service (STS) Management API is an API for managing STS configurations.
   version: 0.0.1
 
+tags:
+  - name: Issuers
+  - name: OAuthClients
+  - name: Users
+
 paths:
   /api/v1/owners/{ownerID}/issuers:
+    get:
+      summary: Gets issuers by owner id
+      operationId: GetOwnerIssuers
```

Comment:
would it be better to call this `ListOwnerIssuers`? same with `clients` and `users`

Reply:
yeah good idea